recipes/sol2/3.x.x/test_package/conanfile.py

```diff
@@ -0,0 +1,21 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.configure()
+        return cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings) or tools.os_info.is_windows:
```

Comment:
```suggestion
        if not tools.cross_building(self.settings):
```

Reply:
thanks , I should not do so much copy and past :-)